src/main/org/firebirdsql/jca/FBTpb.java

```diff
@@ -88,6 +88,30 @@ public void setReadOnly(boolean readOnly) {
     public boolean isReadOnly() {
         return transactionParams.hasArgument(TransactionParameterBuffer.READ);
     }
+
+    /**
+     * Set the auto-commit flag on this TPB.
+     *
+     * @param autoCommit
+     *            If <code>true</code>, this TPB will be set to auto_commit,
+     *            otherwise it won't
+     */
+    public void setAutoCommit(boolean autoCommit) {
```

Comment:
Given bug [JDBC-386: Changes to transaction configuration of a connection are propagated to **all** connections to the same database](http://tracker.firebirdsql.org/browse/JDBC-386) this might have unintended side-effects on the transaction config of other connections.